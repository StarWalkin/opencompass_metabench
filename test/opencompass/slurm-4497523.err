  0%|          | 0/1 [00:00<?, ?it/s]                                       0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [55:17<00:00, 3317.53s/it]100%|██████████| 1/1 [55:17<00:00, 3317.57s/it]
  0%|          | 0/1 [00:00<?, ?it/s]                                       0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:13<00:00, 13.96s/it]100%|██████████| 1/1 [00:13<00:00, 13.96s/it]
