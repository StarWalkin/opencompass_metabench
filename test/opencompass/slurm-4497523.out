/u/zzhang27/.conda/envs/opencompass/bin/python3
08/16 22:01:54 - OpenCompass - INFO - Loading gsm8k_gen: configs/datasets/gsm8k/gsm8k_gen.py
08/16 22:01:54 - OpenCompass - INFO - Loading hf_yi_1_5_9b_chat: configs/models/yi/hf_yi_1_5_9b_chat.py
08/16 22:01:54 - OpenCompass - INFO - Loading example: configs/summarizers/example.py
08/16 22:01:55 - OpenCompass - INFO - Current exp folder: outputs/default/20240816_220155
08/16 22:01:55 - OpenCompass - WARNING - SlurmRunner is not used, so the partition argument is ignored.
08/16 22:01:55 - OpenCompass - INFO - Partitioned into 1 tasks.
launch OpenICLInfer[yi-1.5-9b-chat-hf/gsm8k] on GPU 0
08/16 22:57:12 - OpenCompass - INFO - Partitioned into 1 tasks.
launch OpenICLEval[yi-1.5-9b-chat-hf/gsm8k] on CPU 
dataset    version    metric    mode      yi-1.5-9b-chat-hf
---------  ---------  --------  ------  -------------------
gsm8k      1d7fe4     accuracy  gen                   80.44
08/16 22:57:26 - OpenCompass - INFO - write summary to /scratch/bcwg/Metabench/repo/test/opencompass/outputs/default/20240816_220155/summary/summary_20240816_220155.txt
08/16 22:57:26 - OpenCompass - INFO - write csv to /scratch/bcwg/Metabench/repo/test/opencompass/outputs/default/20240816_220155/summary/summary_20240816_220155.csv
All processes have been started and their PIDs stored in process_ids_gpt4o.txt.
All processes have completed.
